src/components/shared/VirtualizedScrollableGrid.js

```diff
@@ -76,19 +76,20 @@ class VirtualizedScrollableGrid extends React.PureComponent {
   }
 
   updateResponsiveFeatures = debounce(() => {
+    const { theme, columnCount, adaptWidthToElement, width } = this.props;
```

Comment:
destructured it here so the linter can see it being used, also kinda nice for the humans, you can see what `setColumnCount` and `setWidth` actually use from the props directly.

Possibly a `fmt` thing: try to limit (or be more explicit) about the amount of things being passed around, when a function only needs a subset of the object.